recipes/xitorch/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "xitorch" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xitorch/xitorch/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 763d1935f5944589da3cb80a84d2bbdc9f5db193ff9e6893581eff1460e3f54e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - numpy >=1.8.2
+    - python >=3.7
+    - scipy >=1.1.0
+    - pytorch >=1.8
+
+test:
+  imports:
+    - {{ name }}
+  commands:
+    - pip check
+    - pytest -vv --pyargs {{ name }} -k 'not test_equil['
+  requires:
+    - functorch
+    - pip
+    - pytest
+
+about:
+  home: https://xitorch.readthedocs.io/
+  dev_url: https://github.com/xitorch/xitorch
+  summary: Differentiable scientific computing library
+  license: BSD-3-Clause
```

```suggestion
  license: BSD-3-Clause AND MIT
```
Ref: https://github.com/xitorch/xitorch/blob/master/LICENSE